ranger-gcs-plugin/conf/ranger-gcs-security.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Copyright (C) 2020 Google, Inc.
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+       http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+<configuration xmlns:xi="http://www.w3.org/2001/XInclude">
+	<property>
+		<name>ranger.plugin.gcs.service.name</name>
+		<value>gcs</value>
+		<description>
+			Name of the Ranger service containing policies for this gcs connector.
+		</description>
+	</property>
+
+	<property>
+		<name>ranger.plugin.gcs.policy.source.impl</name>
+		<value>org.apache.ranger.admin.client.RangerAdminRESTClient</value>
+		<description>
+			Class to retrieve policies from the source
+		</description>
+	</property>
+
+	<property>
+		<name>ranger.plugin.gcs.policy.rest.url</name>
+		<value>http://policymanagerhost:port</value>
+		<description>
+			URL to Ranger Admin
+		</description>
+	</property>
+
+	<property>
+		<name>ranger.plugin.gcs.policy.rest.ssl.config.file</name>
+		<value></value>
+		<!--value>/etc/hadoop/conf/ranger-policymgr-ssl.xml</value-->
+		<description>
+			Path to the file containing SSL details to contact Ranger Admin
+		</description>
+	</property>
+
+	<property>
+		<name>ranger.plugin.gcs.policy.pollIntervalMs</name>
+		<value>30000</value>
+		<description>
+			How often to poll for changes in policies?
```

Comment:
nit: no need to use a question mark.

Reply:
There is also a question mark in Ranger's built in plugin, I think it's fine?
https://github.com/apache/ranger/blob/master/hdfs-agent/conf/ranger-hdfs-security.xml#L56